src/group_theory/perm/cycle_type.lean

```diff
@@ -239,6 +296,29 @@ def partition (σ : perm α) : partition (fintype.card α) :=
   parts_sum := by rw [sum_add, sum_cycle_type, multiset.sum_repeat, nsmul_eq_mul,
     nat.cast_id, mul_one, nat.add_sub_cancel' σ.support.card_le_univ] }
 
+@[simp]
```

Comment:
Should this be `@[simp]`? I think I would rather see it added manually to the simp-set when needed.

Reply:
Ok